### PR TITLE
Update R24, Adding R25 and R26

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Update R24 (close it)
- Add R25, R26

### CODE REVIEW

**Comment by jeffvg** [comment]
should we do anything with announcement.json?

**Comment by emclaug2** [comment]
> should we do anything with announcement.json?

I've updated the banner now to include and announcement for R25.

File: src/data/doc-it/Announcement.ts [hunk 1]
```diff
@@ -10,10 +10,10 @@ type Announcement = {
 
 export const announcement: Announcement = {
     announcement: {
-        id: 1,
-        bannerContent: "We've related updated to <a href='/release-notes'> R24 </a>! Checkout our release presentation on <a href='https://www.youtube.com/watch?v=jFbutosoJ3Q' target='_blank'> Youtube </a>",
-        startDate: "2022-04-15",
-        endDate: "2022-04-25",
-        devOnly: true
+        id: 2,
+        bannerContent: "We've released updates for R25!  See our release notes for <a href='/release-notes'>details</a>.  ",
```
Comment (huayunh) [inline src/data/doc-it/Announcement.ts:14]: "R25" -> "Q2 2022" will make more sense to people outside Brightlayer UI

File: src/data/doc-it/R25Roadmap.ts [hunk 2]
```diff
@@ -26,53 +26,43 @@ type RoadmapBucket = {
...
-                description: 'Updated information on proper usage of icons',
-                year: CURRENT_YEAR,
-                quarter: CURRENT_QUARTER,
-                status: 'backlog',
-                author: 'Brightlayer UI',
+                "name": "Icon Guidelines",
+                "description": "Updated information on proper usage of icons",
+                "year": 2022,
+                "quarter": "Q3",
+                "status": "backlog",
```
Comment (huayunh) [inline src/data/doc-it/R25Roadmap.ts:38]: "in progress"

File: src/data/doc-it/R25Roadmap.ts [hunk 3]
```diff
@@ -26,53 +26,43 @@ type RoadmapBucket = {
...
-        framework: ['all'],
+        name: 'Figma Resources',
+        description: 'Updating and adding designs in Figma',
+        type: 'design',
         items: [
             {
-                name: 'Dashboard Data Viz Components',
-                description: 'Develop reusable visualization components for embedding in dashboards',
+                name: 'Accessible themes',
+                description: 'Designing accessible theme standards',
```
Comment (huayunh) [inline src/data/doc-it/R25Roadmap.ts:50]: Can we unify the verb tenses? "Designing" "Updated" "Implement"

**Review by huayunh — DISMISSED** [review]

File: src/data/doc-it/R25Roadmap.ts (hunk 2 above)
Comment (emclaug2) [reply src/data/doc-it/R25Roadmap.ts:38]: Updated

**Comment by emclaug2** [comment]
@huayunh Ready for another round; thanks for the review

**Pull request merged**